Implement a KISS version of pub/sub (#4118)
tenzir/tenzir: web/docs/operators/subscribe.md

---
sidebar_custom_props:
  operator:
    source: true
---

# subscribe

> Subscribes to events at a Tenzir Node. The dual to [`publish`](publish.md).

Reviewer: ```suggestion
Subscribes to events from a channel with a topic. The dual to [`publish`](publish.md).
```

## Synopsis

```
subscribe [<topic>]
```
## Description

> The `subscribe` operator subscribes to events at a Tenzir Node from a channel

Reviewer: ```suggestion
The `subscribe` operator subscribes to events from a channel
```
